File: JavaScript/bubblesort.js
const bubbleSort = arr => {
    let swapCount = 1;
    
    while (swapCount > 0) {
      swapCount = 0;
      for (let i = 0; i < arr.length - 1; i++) {
        if (arr[i] > arr[i + 1]) {
          let tmp = arr[i];
          arr[i] = arr[i + 1];
          arr[i + 1] = tmp;
          swapCount++;
        }
      }
    }
    return arr;
  };

export default bubbleSort;

File: Python/bubblesort.py
# bubble sort module

def bubbleSort(arr):
    swaps = 1

    while(swaps > 0):
        swaps = 0
        for i in range(0, len(arr) - 1):
            if arr[i]  > arr[i+1]:
                tmp = arr[i]
                arr[i] = arr[i+1]
                arr[i+1] = tmp
                swaps +=1
    
    return arr
